quickwit/quickwit-metastore/src/tests/mod.rs

```diff
@@ -363,11 +377,22 @@ macro_rules! metastore_test_suite {
                 $crate::tests::split::test_metastore_stage_splits::<$metastore_type>().await;
             }
 
-            /// Shard API tests
+        }
+    };
+}
 
+#[macro_export]
```

Comment:
I had to remove the shard_tests from the common tests because of import issues caused by using sqlx. I tried adding an if condition based on `#[cfg(feature="postgres")]` but I quickly realized that the tests would be run for the `FileBackedIndex` even in that case. I am open to other ideas to make this better.